server/src/main/java/com/cloud/resource/ResourceManagerImpl.java

```diff
@@ -2178,7 +2179,7 @@ private Host createHostAndAgentDeferred(final ServerResource resource, final Map
                     // added, no
                     // need to add
                     // again
-                    s_logger.debug("Found the host " + host.getId() + " by guid: " + firstCmd.getGuid() + ", old host reconnected as new");
+                    s_logger.debug(String.format("Found host [id: %d, name: %s] by guid %s, old host reconnected as new", host.getId(), host.getName(), firstCmd.getGuid()));
```

Comment:
@GabrielBrascher We can log the entire object here, as it has toString() implemented.